oauth2client/client.py

```diff
@@ -229,7 +225,7 @@ def authorize(self, http):
             http: httplib2.Http, an http object to be used to make the refresh
                   request.
         """
-        _abstract()
+        raise NotImplementedError
```

Comment:
I prefer `()` at the end of `raise NotImplementedError`, but if you feel strongly it's also fine to leave these as they are.